src/DynamoPackages/PkgAssemblyLoadContext.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace DynamoPackages
+{
+    internal class PkgAssemblyLoadContext : AssemblyLoadContext
+    {
+        private AssemblyDependencyResolver _resolver;
+        public PkgAssemblyLoadContext(string name, string pluginPath, bool unloadable = true) : base(name, unloadable)
+        {
+            _resolver = new AssemblyDependencyResolver(pluginPath);
```

Comment:
somehow I think we'd also want to include the `/extra` folder, maybe we can use two AssemblyDependencyResolvers?